enos/enos-modules.hcl

```diff
@@ -307,3 +307,10 @@ module "verify_seal_type" {
 
   vault_install_dir = var.vault_install_dir
 }
+
+module "vault_verify_billing_start_date" {
+  source = "./modules/vault_verify_billing_start_date"
+
```

Comment:
We should probably set the cluster port here, e.g. https://github.com/hashicorp/vault/blob/main/enos/enos-modules.hcl#L234

Reply:
added